src/dmgbuild/licensing.py

```diff
@@ -382,87 +158,119 @@
     ),
 }
 
-
-class LPicResource(Resource):
-    def __init__(self, res_id, res_name, default_lang, lpic, res_attrs=0):
-        data = []
-        data.append(struct.pack(b'>HH', default_lang, len(lpic)))
-        for lang, rid, two_byte in lpic:
-            data.append(struct.pack(b'>HHH', lang, rid, int(two_byte)))
-        super().__init__(b'LPic', res_id, res_name, b''.join(data), res_attrs)
-
-
-def get_encoder_name(locale):
-    if locale not in region_codes:
-        raise Exception("Cannot determine region code for locale '%s'" % locale)
-    region_code = region_codes[locale]
-
-    if region_code not in script_codes:
-        raise Exception("Cannot determine script code for locale '%s'" % locale)
-    script_code = script_codes[region_code]
-
-    if script_code not in encodings_map:
-        raise Exception("Cannot determine Python encoder name for locale '%s' - "
-                        "encode the string data manually as a byte array instead" % locale)
-    return encodings_map[script_code]
+ufifrezXMLtemplate = {
+    'LPic': [
+        {
+            'Attributes': '0x0000',
+            'Data': b'\x00\x00\x00\x02\x00\x00\x00\x00\x00\x00\x00\x00\x00\x04\x00\x00',
+            'ID': '5000',
+            'Name': ''
+        }
+    ],
+    'STR#': [
+        {
+            'Attributes': '0x0000',
+            'Data': b'\x00\x06\rEnglish test1\x05Agree\x08Disagree\x05Print\x07Save...zIMPORTANT - Read this License Agreement carefully before clicking on the "Agree" button. By clicking on the "Agree" button, you agree to be bound by the terms of the License Agreement.',  # noqa: E501
```

Comment:
Two details here: 

1. Is the "English test1" portion intentional? 
2. When I mount the disk image created using the `examples/settings.py` script, the acknowledgement text is truncated:
<img width="758" alt="Screen Shot 2022-06-22 at 8 58 58 am" src="https://user-images.githubusercontent.com/37345/174921352-11280427-1ea6-4925-9fc1-879301183d3c.png">

I've tested this in both en_AU and en_UK configurations

Reply:
oh, I dont know, will make the default acknowledge message smaller...